akka-cluster-client-grpc-scala/README.md

```diff
@@ -0,0 +1,49 @@
+# Cluster Client with gRPC transport
+
+The purpose of this example is to illustrate how the deprecated
+[Akka Cluster Client](https://doc.akka.io/docs/akka/2.5/cluster-client.html)
+can be replaced by [Akka gRPC](https://doc.akka.io/docs/akka-grpc/current/index.html).
+
+The example is intended to be copied and adjusted to your needs. It will not be
+provided as a published artifact.
+
+The example is still using an actor on the client side to have an API that is very close
+to the original Cluster Client. The messages this actor can handle correspond to the
+[Distributed Pub Sub](https://doc.akka.io/docs/akka/current/distributed-pub-sub.html)
+messages on the server side, such as `ClusterClient.Send`.
+
+The `ClusterClient` actor delegates those messages to the gRPC client and on the
+server side those are translated and delegated to the destination actors that
+are registered via the `ClusterClientReceptionist` in the same way as in the original.
+
+The application specific messages are wrapped and serialized with Akka serialization.
```

Comment:
"Which means that care must be taken to keep wire compatibility when changing any messages used between the client and server." Or something like that?

Reply:
Instead of this readme I will link to https://doc.akka.io/docs/akka/2.6/cluster-client.html#migration-to-akka-grpc so that we have the text in one place.

I'll add this clarification over there.